types/express-ws-routes/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for express-ws-routes 1.1
+// Project: https://github.com/amekkawi/express-ws-routes
+// Definitions by: Gregor Stamać <https://github.com/gstamac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Express as exExpress, Request, NextFunction, Application } from 'express';
+import * as WebSocket from 'ws';
+import { Server as httpServer } from 'http';
+import { Router as coreRouter } from 'express-serve-static-core';
+
+declare function expressWsRoutes(): expressWsRoutes.Express;
+
+declare namespace expressWsRoutes {
+	type ClientInfo = { origin: string; secure: boolean; req: Request };
```

Comment:
This should be an interface and you shouldn't need to have an exception in the dtslint.